### PR TITLE
WIP: Starting a single control trace.

### PR DESCRIPTION
**Pull request by Compton-NIST**
# Committer Notes

See #141 

### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/oscal-content/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/oscal-content/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?

### CODE REVIEW

File: examples/trace-a-single-control/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Assessment Layer
+
+5. Assessment Plan
+6. Assessment Result
+7. Plan of Action and Milestone
+
+![Graphic displaying the connections between OSCAL models](diagrams/Concept.drawio.svg)
+
+One concept that is important to understand is the meaning of a **resolved profile**.  In this example, the profile produces a subset of controls from the NIST 800-53 Rev 5 catalog, specifically we only select one control: AC-8.  Until this profile is resolved, a catalog does not exist that only contains the subset.  For this example, the profile is considered resolved, and the catalog model would be the result of this transformation.
```
Comment (aj-stein-nist) [inline examples/trace-a-single-control/README.md:31]: Minor and non-blocking: perhaps be explicit with the verb a catalog+profile->a new catalog?
```suggestion
One concept that is important to understand is the meaning of a **resolved (profile) catalog**.  In this example, a profiler resolver tool will use a profile to produce a new catalog with a subset of controls from the NIST 800-53 Rev 5 catalog. Specifically, we only select one control: AC-8.  Until this profile is resolved, a catalog does not exist that only contains the subset.  For this example, the profile is considered resolved, and the catalog model would be the result of this transformation.
```

**Review by aj-stein-nist — COMMENTED** [review]
I apologize, you had asked me to look at this long ago and had not followed up. I know it is still WIP, but one minor question.

File: examples/trace-a-single-control/README.md (hunk 1 above)
Comment (Compton-NIST) [reply examples/trace-a-single-control/README.md:31]: No apology necessary! Good feedback!

File: examples/trace-a-single-control/yaml/system-security-plan.yaml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        - title: System Assessment
+          description: This system handles development information pertaining to audit and assessment events to demonstrate the OSCAL workflow.
+          categorizations:
+            - system: https://doi.org/10.6028/NIST.SP.800-60v2r1
+              information-type-ids:
+                - C.3.5.1
+          confidentiality-impact:
+            base: fips-199-low
+          integrity-impact:
+            base: fips-199-low
```
Comment (aj-stein-nist) [inline examples/trace-a-single-control/yaml/system-security-plan.yaml:37]: I actually had to double check 800-60 Revision 2 and lazily looked up this PR to see what the structure was and translate it back into XML for another PR. FYI/FYE, the actually recommended (`base`) for `C.3.5.1` is Low-Moderate-Low.

```suggestion
            base: fips-199-moderate
```

If you wanted to override and say well the book says is moderate, but my AO, ISSO, organizational leadership, what have you, said it is different, you could do it.

```yaml
          integrity-impact:
            base: fips-199-moderate
            selected: fips-199-low
```

**Review by aj-stein-nist — COMMENTED** [review]
Minor thing I caught when leveraging your sample for a quick sanity check. :-)

**Comment by aj-stein-nist** [comment]
Given the conversation in https://github.com/usnistgov/oscal-content/issues/141#issuecomment-1412309177 that was discussed a little while back, I am going to close this out for now. We can reopen as we see fit when the work comes back into focus in the next sprint or a subsequent one.

**Pull request closed without merging**